python/ray/tests/test_object_manager.py

```diff
@@ -149,8 +149,11 @@ def set_weights(self, x):
             pass
 
     actors = [
-        Actor._remote(args=[], kwargs={}, resources={str(i % num_nodes): 1})
-        for i in range(100)
+        Actor._remote(
+            args=[],
+            kwargs={},
+            num_cpus=1 / 100,
```

Comment:
What was the reason for this change? I guess there weren't enough CPU resources to create the actors before?
```suggestion
            num_cpus=0.01,
```

Reply:
Yup that was the issue